<launch>

	<include file="$(find visor)/launch/camera_setting.xml" />

	<arg name="display_image" default="true" />
	
	<!-- DISPLAY IMG -->
	<node if="$(arg display_image)" name="image_view" pkg="image_view" type="image_view" respawn="true" output="screen">
		<remap from="image" to="/resized_img/image_raw"/>
		<param name="autosize" value="true" />
	</node>	
	
</launch>
